MSliceMainView.py

```diff
@@ -0,0 +1,221 @@
+#This file in its current state only exists for reference
+#import abc
```

Comment:
You are not using this module, so delete the line and all associated decorators.